src/like-service/app/Http/Middleware/EncryptCookies.php

```diff
@@ -12,6 +12,6 @@ class EncryptCookies extends Middleware
      * @var array
      */
     protected $except = [
-        //
+        'jwt'
```

Comment:
[DISCUSS] I think we should just remove this middleware

Reply:
Discussed. Removing would cause further issues